vizro-core/docs/pages/development/documentation-style-guide.md

```diff
@@ -0,0 +1,143 @@
+# Vizro documentation style guide
+
+This is the style guide we apply to the [Vizro documentation](https://vizro.readthedocs.io/en/stable/).
+
+We ask anyone kind enough to contribute documentation changes to follow this style for consistency and simplicity.
+
+What follows is a set of lightweight guidelines rather than rules. There are always edge cases and exceptions, and if it's not obvious what the style should be, consult the [Microsoft style guide](https://docs.microsoft.com/en-gb/style-guide/welcome/) for an example of good practice. We also use the [INCITS Inclusive Terminology Guidelines](https://standards.incits.org/apps/group_public/download.php/131246/eb-2021-00288-001-INCITS-Inclusive-Terminology-Guidelines.pdf).
+
+## Vizro lexicon
+
+The names of our products are **Vizro** and **Vizro-AI**.
+
+We refer to other products using their preferred capitalization. For example, Dash and Plotly are always capitalized, except where given as Python package names `dash` and `plotly`.
+
+Vizro components are named using lower case:
+
+> "Here is a guide to using containers..."
+
+Use code font when referring to the component as a class or object:
+
+> "To add a `Container` to your page..."
+
+## Bullets
+* Capitalize the first word, and end the bullet with a period.
+* Don't use numbered bullets except for a sequence of instructions, or where you have to refer back to one of them in the text (or a diagram).
+
+## Call out boxes
+
+Keep the amount of text, and the number and variety of callouts used, to a minimum. There is a [broad set available](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) for use in the Vizro docs, but we prefer to limit usage mostly to the following:
+
+!!!note "note"
+
+     For notable information
+
+!!!warning "warning"
+
+     To indicate a potential gotcha.
+
+!!!example "example"
+
+     For example code
+
+
+Boxes can be made collapsible, and rendered initially collapsed or expanded.
```

Comment:
```suggestion
Boxes can be made collapsible, and if done so preferably should be rendered initially collapsed.
```
Point for discussion - but I think it's a good light rule. Make things collapsible if they are crowding the page, and hide them initially, do not make things collapsible that should always be visible